content/en/blog/falco-talon-v0-1-0/index.md

```diff
@@ -0,0 +1,421 @@
+
+A frustation remained after all. With Falco, we have an amazing tool to detect suspicious events in our Linux hosts, VM and Kubernetes clusters, with Falcosidekick, we can easily notify our Dev/Secops, index the alerts in some SIEM, etc. But a last piece was missing: how to react to these events? 
+
+    With the integrations of well known FaaS in Falcosidekick, we started a [series of blog posts](https://falco.org/blog/falcosidekick-response-engine-part-1-kubeless/) to show how to create from scratch what we call a "response engine". All these systems are modular, flexible, robust, but they all require a lot of developments, to deal with the Falco payload format, the errors, the retries, the authentication to the API (AWS, Kubernetes Control Plane), the logs, the metrics, etc. Not all users and companies have the skills and/or the budgets to deal with that.
+
+    **This is exactly to answer these needs, we designed and created [`Falco Talon`](https://github.com/falco-talon/falco-talon), of which [the first version is officially out](https://github.com/falco-talon/falco-talon/releases/tag/v0.1.0).**
+
+## What is Falco Talon?
+
+`Falco Talon` is a *Response Engine* for managing threats in Kubernetes clusters. It enhances the solutions proposed by the Falco community with a *no-code tailor-made* solution. With easy rules, you can react to events from Falco in milliseconds.
+
+### Why having created Falco Talon?
+
+Over the years, the Falco community proposed different methods to react to the Falco Events, what we call a *response engine*. 
+
+All these methods rely on a 3rd party FaaS (*Function as a Service*) and come with drawbacks, all actions must be developped by the users to manage:
+
+* The errors
+* The Falco event format
+* The authentication
+* The K8s SDK complexity
+* The security
+* The upgrades of the dependencies
+* Latency
+* Complexity to manage sequential actions
+* Intrication between the function and the configuration
+
+**This is why we started to develop a custom solution specifically built for Falco: `Falco Talon`**:
+
+* Tailor made for the Falco events
+* Easy to define rules
+* No-code implementation for end-users
+* UX close to Falco with the rules (yaml files with append, override mechanisms)
+* Allow to set up sequential actions to run
+* Structured logs (with a trace id)
+* OTEL/Prometheus Metrics and OTEL Traces
+
+### What is it good for?
+
+* React in real-time to the Falco Events
+* Allow fine granularity to match the events to react to
+* Responding to default rules with specific overrides
+
+### What is it not (yet?) good for?
+
+* Complex reaction worflows with conditions between the steps
+* Act outside a Kubernetes context (no ssh-through actions like Ansible)
```

Comment:
What does it mean? Does it mean that I cannot install this yet on a node without k8s? In that case it should be better specified for people that do not know what ssh-through actions are.